postActions/etcd.yml

```diff
@@ -0,0 +1,21 @@
+config:
+  runStrategy:
+    runs: 1
```

Comment:
Not sure, will give that a try today!

Reply:
I think the way I have it coded, the multiple runs won't work properly. I can definitely change it if necessary. But the output returns all the output from all runs so if we do 10 runs and and the first 5 fail and then we finally get 3 pods; it will still return "ERROR" in the response so the current implementation will think it failed. Thoughts on what to do here?

Reply:
Maybe keep checking the status and break once the condition is met and if not then timeout and fail after the set number of runs.

Reply:
Also think we need exit 1 to report the scenario failure and skip running the next scenarios since that might have a dependency on the failed component or it might affect the cluster more.

Reply:
I can see two "modes" for Kraken:
1. Run a list of scenarios.  At the end, is the cluster healthy or no?
2. Run Kraken scenarios in daemon mode and see if the cluster can heal itself and stay healthy over time.   This would be useful in our reliability clusters (/cc: @wabouhamad )

Should we add a config option for exit/no exit?   An option to just keep dealing damage even if the cluster is in a suspect state?

Reply:
I agree with the 2 scenarios and I feel like I have one covered currently. 
For the first scenarios I'll keep track of the failed post action scenarios and at the end give a status. For the second one I could either use the exit/no exit option in addition to the failed scenarios. For the no exit option I will just keep track of the failed scenarios and print them out so the user knows the failures during the daemon run? 

I definitely like the idea of exit/no exit option similar to the cerberus go/no go status. Would this be part of this pull request or should we make it separate? I think this would help us decide on some of the decisions we are talking about

Reply:
Cerberus signal is going to false leading to Kraken exiting 1 at the end of the iteration even in daemon mode if the components haven't recovered after the time specified in the scenario. Think that should be the behavior of Kraken in both non daemon and daemon mode for us to know when things are failing, also it might drive the cluster down the cluster down the hill if kraken keeps injecting failures even when things don't recover but that would be an interesting thing to do as well I guess.